src/EditCell.tsx

```diff
@@ -1,37 +1,38 @@
 import { useState, useCallback } from 'react';
+import { createPortal } from 'react-dom';
 import { css } from '@linaria/core';
 
-import EditorContainer from './editors/EditorContainer';
+import { useMouseDownOutside } from './hooks';
 import { getCellStyle, getCellClassname } from './utils';
-import type { CellRendererProps, SharedEditorProps, Omit } from './types';
+import type { CellRendererProps, SharedEditorProps } from './types';
 
 const cellEditing = css`
   padding: 0;
 `;
 
-const cellEditingClassname = `rdg-cell-editing ${cellEditing}`;
+// TODO: remove 1 of the two classes
+const cellEditingClassname = `rdg-cell-editing rdg-editor-container ${cellEditing}`;
```

Comment:
WDYT? We have some internal code depending on both class names at the moment.

Reply:
I think we can remove one if possible